coremltools/converters/mil/frontend/torch/test/test_executorch_e2e.py

```diff
@@ -92,6 +94,139 @@ def forward(self, a, x, b):
             backend=("mlprogram", "fp16"),
         )
 
+    def test_softmax(self):
```

Comment:
what is the reason some model testing fp32 while some testing fp16?

Reply:
I prefer to use fp16 whenever possible

For now, mobilebert is the only fp32 one, since it has a bit too large numerical error, and even with fp32 `rtol` still needs to be tuned